eng/integration-tests.yml

```diff
@@ -31,12 +31,31 @@ steps:
       unzip /tmp/terraform.zip -d /tmp
       mkdir -p ~/bin
       export PATH="~/bin:$PATH"
-      export ARM_SUBSCRIPTION_ID=$(AZURE_SUBSCRIPTION_ID)
-      export ARM_CLIENT_ID=$(AZURE_CLIENT_ID)
-      export ARM_CLIENT_SECRET=$(AZURE_CLIENT_SECRET)
-      export ARM_TENANT_ID=$(AZURE_TENANT_ID)
       export GO111MODULE=on
       make test-cover
       goveralls -coverprofile=cover.out -service=azure-devops
       make destroy-sb
+    env:
+      ARM_SUBSCRIPTION_ID: $(go-eh-azure-subscription-id)
+      ARM_CLIENT_ID: $(go-eh-azure-client-id)
+      ARM_CLIENT_SECRET: $(go-eh-azure-client-secret)
+      ARM_TENANT_ID: $(go-eh-azure-tenant-id)
     displayName: 'Run Integration Tests'
+  - script: |
+      set -e
+      go test -race -v -coverprofile=coverage.txt -covermode atomic ./... 2>&1 | go-junit-report > report.xml
```

Comment:
Make test-cover should include this, right?

Reply:
Correct.  Be sure to destroy after.

Reply:
ok good, removed and calling destroy correctly now.